integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpcds/TpcdsLikeSpark.scala

```diff
@@ -0,0 +1,4628 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.tests.tpcds
+
+import org.apache.spark.sql.{DataFrame, DataFrameWriter, Row, SparkSession}
+import org.apache.spark.sql.types.{DateType, DoubleType, IntegerType, LongType, StringType, StructField, StructType}
+
+// scalastyle:off line.size.limit
+
+case class Table(
+    name: String, // also the base name for the data
+    partitionColumns: Seq[String],
+    schema: StructType) {
+
+  private[this] def path(basePath: String) =
+    basePath + "/" + name + ".dat"
+
+  def readCSV(spark: SparkSession, basePath: String): DataFrame =
+    spark.read.option("delimiter", "|")
+        .schema(schema)
+        .csv(path(basePath))
+
+  def setupCSV(spark: SparkSession, basePath: String): Unit =
+    readCSV(spark, basePath).createOrReplaceTempView(name)
+
+  def setupParquet(spark: SparkSession, basePath: String): Unit =
+    spark.read.parquet(path(basePath)).createOrReplaceTempView(name)
+
+  def setupOrc(spark: SparkSession, basePath: String): Unit =
+    spark.read.orc(path(basePath)).createOrReplaceTempView(name)
+
+  private def setupWrite(
+      spark: SparkSession,
+      inputBase: String): DataFrameWriter[Row] = {
+    val tmp = readCSV(spark, inputBase)
+        .write
+        .mode("overwrite")
+    tmp
+//    if (partitionColumns.isEmpty) {
```

Comment:
need this?

Reply:
As I mentioned in the PR that I disabled the partition support.  This is something that databricks was doing, so I wanted to capture it, but for small amounts of data it just ends up making thousands of small files.  I wanted to leave the code there so that we could enable it in the future if we got serious about benchmarking using tpc-ds.

Reply:
ah ok sorry I missed it. Thanks